tests/common-tests.h

```diff
@@ -55,7 +55,20 @@ inline namespace boost {
 #endif
 #endif // HAVE_BOOST
 
-
+// Firebird returns string that represents entire byte-size(not character size) of the column( i.e. CHAR(10) NONE - 10bytes, CHAR(10) UTF8 - 40bytes )
+// MySql returns right trimmed value by default or space padded up to column character size when PAD_CHAR_TO_FULL_LENGTH
+// PostgreSQL returns space padded version up to actual column character size (not byte size )
+// SQLite has TEXT affinity for all char types so it is not an issue here - macro bellow will match correctly
+// TODO: comments for other backends(DB, ODBC)
+//This macro will check result string from column defined as fixed char
+//It will check only bytes up to original string size.
+//If result string is bigger than original string then all remaining chars must be spaces so if any non-space char is found it will fail .
+#define CHECK_CHAR( STR, ORIG_STR ) \
```

Comment:
The macro name is too generic, I'd call it `CHECK_EQUAL_PADDED()` or something like this.

Also, from stylistic point of view, it's weird to use ALL CAPS for the macro arguments, they usually use the same convention as normal variables (i.e. I'd call them `padded_str` and `expected_str` perhaps).